docs/documentation/upgrading/topics/migrate_db.adoc

```diff
@@ -5,6 +5,11 @@
 {project_name} can automatically migrate the database schema, or you can choose to do it manually. By default the
 database is automatically migrated when you start the new installation for the first time.
 
+[NOTE]
+====
+Before you migrate the database, shut down all nodes.
```

Comment:
I think this needs more context as users might assume they need to shut down the database nodes. See below for a suggestion to clarify it.

```suggestion
Before you migrate the database, shut down all {project_name} nodes running the old version of {project_name}.
```